### PR TITLE
New package: neard-0.18

### PR DESCRIPTION
**Pull request by JamiKettunen**
I've verified the [PN544 NFC chip in my phone](https://gitlab.com/msm8998-mainline/linux/-/commit/41fa0860) functions just fine via this NFC daemon.

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/neard/files/neard/run [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+sv check dbus >/dev/null || exit 1
+[ -f conf ] && . ./conf
+exec /usr/libexec/nfc/neard ${ARGS:--n}
```
Comment (paper42) [inline srcpkgs/neard/files/neard/run:4]: If -n means don't fork to background, we should set it all the time even if ARGS is set and we usually call this variable OPTS.

File: srcpkgs/neard/files/neard/run [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+sv check dbus >/dev/null || exit 1
+[ -f conf ] && . ./conf
```
Comment (paper42) [inline srcpkgs/neard/files/neard/run:3]:
```suggestion
[ -r conf ] && . ./conf
```

File: srcpkgs/neard/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+makedepends="glib-devel libnl-devel"
+short_desc="Near Field Communication (NFC) manager"
+maintainer="Jami Kettunen <jami.kettunen@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/linux-nfc/neard"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=c013e3115315e2e11d16b5f0f1c8ce9afbcdba26800aad91245fd9b4a5dc938b
+
+pre_configure() {
+	./bootstrap # generate configure script
```
Comment (paper42) [inline srcpkgs/neard/template:17]:
```suggestion
	autoreconf -fi
```

File: srcpkgs/neard/template [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+revision=1
+build_style=gnu-configure
+configure_args="--disable-systemd --enable-tools"
+hostmakedepends="autoconf autoconf-archive automake libtool pkg-config"
+makedepends="glib-devel libnl-devel"
+short_desc="Near Field Communication (NFC) manager"
+maintainer="Jami Kettunen <jami.kettunen@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/linux-nfc/neard"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/neard/template:13]:
```suggestion
distfiles="https://github.com/linux-nfc/neard/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/neard/template [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+pre_configure() {
+	./bootstrap # generate configure script
+}
+
+post_install() {
+	vsv neard
+}
+
+neard-devel_package() {
+	depends="neard>=${version}_${revision}"
```
Comment (paper42) [inline srcpkgs/neard/template:25]:
```suggestion
	depends="${sourcepkg}>=${version}_${revision}"
```

File: srcpkgs/neard/files/neard/run (hunk 2 above)
Comment (JamiKettunen) [reply srcpkgs/neard/files/neard/run:3]: Any reason this is better? It seems ` -f` and `-r` are mixed in `run` files across the tree. I mean I suppose we do have to be able to read it to source it.

File: srcpkgs/neard/template (hunk 4 above)
Comment (JamiKettunen) [reply srcpkgs/neard/template:13]: If you don't mind I'll switch the URLs to https://git.kernel.org/pub/scm/network/nfc/neard.git; I'd use `${KERNEL_SITE}/network/nfc/neard-${version}.tar.xz` but they only have "pre-configured" archives up to v0.16 :/

Comment (JamiKettunen) [reply srcpkgs/neard/template]: Ah nvm I just checked the README again, development moved away from kernel.org repo.

File: srcpkgs/neard/files/neard/run (hunk 2 above)
Comment (paper42) [reply srcpkgs/neard/files/neard/run]: That's interesting, all services in base use -r, but it doesn't seem universal outside of base.

**Pull request merged**